be/tool/beck.c

```diff
@@ -616,10 +618,12 @@ int main(int argc, char **argv)
 		   M0_STRINGARG('a', "stob domain path",
 			   LAMBDA(void, (const char *s) {
 				   beck_builder.b_stob_path = s;
+				   b_stob_path_length = sizeof(*s);
```

Comment:
This is still wrong.
Now, the `sizeof(*s)` is sizeof (char), which is one byte.

Reply:
I think we have to use strlen() here.

Reply:
Okay sure.